src/app/item-page/simple/field-components/specific-field/geospatial/geospatial-item-page-field.component.ts

```diff
@@ -0,0 +1,86 @@
+import { NgIf } from '@angular/common';
+import {
+  Component,
+  Input,
+  OnInit,
+} from '@angular/core';
+import { TranslateModule } from '@ngx-translate/core';
+
+import { Item } from '../../../../../core/shared/item.model';
+import {
+  hasValue,
+  isNotEmpty,
+} from '../../../../../shared/empty.util';
+import { GeospatialMapComponent } from '../../../../../shared/geospatial-map/geospatial-map.component';
+import { MetadataFieldWrapperComponent } from '../../../../../shared/metadata-field-wrapper/metadata-field-wrapper.component';
+import { ItemPageFieldComponent } from '../item-page-field.component';
+
+@Component({
+  selector: 'ds-geospatial-item-page-field',
+  templateUrl: './geospatial-item-page-field.component.html',
+  imports: [
+    MetadataFieldWrapperComponent,
+    GeospatialMapComponent,
+    TranslateModule,
+    NgIf,
+  ],
+  standalone: true,
+})
+/**
+ * This component can be used to represent metadata on a simple item page.
+ * It is the most generic way of displaying metadata values
+ * It expects 4 parameters: The item, a separator, the metadata keys and an i18n key
+ */
+export class GeospatialItemPageFieldComponent extends ItemPageFieldComponent implements OnInit {
+
+  /**
+   * The item to display metadata for
+   */
+  @Input() item: Item;
+
+  /**
+   * Label i18n key for the rendered metadata
+   */
+  @Input() label: string;
+
+  /**
+   * List of fields to parse for WKT points
+   */
+  @Input() pointFields = ['dcterms.spatial'];
+
+  /**
+   * List of fields to parse for bounding box GeoJSON
+   */
+  @Input() bboxFields = ['gnd.coordinates.bbox'];
```

Comment:
This field doesn't exist in DSpace?  Is it a metadata field, or something else?

Reply:
updated to dcterms.spatial (and ensured the rendering can support bbox and points without conflict) since there is no other appropriate field to use as a default. We could also remove a default and force it to be determined only by user input if there is concern about `dcterms.spatial` usage, though the DCMI documentation appears to support this usage which is why I picked it.